### ISSUE TITLE
Docker Image Creation Issue in Windows 10

### ISSUE THREAD

**Issue by SatishMHiremath** [dotnet/core#5184]
Problem encountered on https://dotnet.microsoft.com/learn/aspnet/microservice-tutorial/docker-image
Operating System: windows

Provide details about the problem you're experiencing. Include your operating system version, exact error message, code sample, and anything else that is relevant.

C:\Users\UserName\workspace\ApplicationName\Application>docker image build -t appln .
Sending build context to Docker daemon  20.14MB
Step 1/4 : FROM openjdk:8
8: Pulling from library/openjdk
4612f6d0b889: Pull complete                                                                                                                                                 c3aff4450246: Pull complete                                                                                                                                                 5af76ffebd6b: Pull complete                                                                                                                                                 88c150df4792: Pull complete                                                                                                                                                 25f43679dd83: Pull complete                                                                                                                                                 301e537a5004: Pull complete                                                                                                                                                 748b1d117c33: Pull complete                                                                                                                                                 add34b581bb7: Pull complete                                                                                                                                                 Digest: sha256:202f23811f6cf23df64b2633f42bd46f8a597bdda26b6ea07259379a9da39c39
Status: Downloaded newer image for openjdk:8
 ---> ab8aa7396023
Step 2/4 : COPY ./target/appln.jar app.jar
hcsshim::PrepareLayer - failed failed in Win32: Incorrect function. (0x1)

**Comment by mairaw** [comment]
@nishanil @shirhatti can you help with this issue?

**Comment by shirhatti** [comment]
Closing as a dupe of https://github.com/dotnet/core/issues/3798

tl;dr: It's an upstream docker issue with some known work-arounds
